ttnn/cpp/ttnn/operations/eltwise/ternary_backward/device/ternary_backward_op.cpp

```diff
@@ -26,9 +28,9 @@ std::vector<Tensor> _addcmul_bw(
     const MemoryConfig& output_mem_config) {
     std::vector<Tensor> grad_tensor;
     grad_tensor.emplace_back(grad);
```

Comment:
emplace_back doesn't change anything in this case. copy constructor will be called.